From f657c3872777ce2af004a3fe97cc00f27b6adf13 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 30 Oct 2012 13:07:45 +0800
Subject: [PATCH 089/310] ENGR00230773 Wifi: add help url to setting menu

Add overlay item for help_url_wifi. This sort items should be overwritten
in product to guide user how to use.Like nexus 7, it point to Nexus's help
url in support.google.com. Here only show how to overwriten thus url.Like bt
,battery etc can follow this by who doing product.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 .../packages/apps/Settings/res/values/strings.xml   | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml

diff --git a/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml b/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml
new file mode 100644
index 0000000..f240913
--- /dev/null
+++ b/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2007 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <!--  Help URLs for some screens. Not specified here. Specified in product overlays --><skip/>
+    <!-- Help menu label [CHAR LIMIT=20] -->
+    <!-- Help URL, WiFi [DO NOT TRANSLATE] -->
+    <string name="help_url_wifi" translatable="false">http://support.google.com/wifi</string>
+</resources>
-- 
1.8.0

